### ISSUE TITLE
termux-open-url file://mnt/sdcard/fish.html

### ISSUE THREAD

**Issue by yesco** [termux/termux-app#136]
Great terminal, finally working emacs!

Creating local files, I'd like to open them in chrome from the command line.
However, the command above says "Cannot display PDF"!

I'm using nexus 7 upgraded to 5.1 and using chrome I can type the same url and it works.

Also, I created the file on the sdcard, as the home directory isn't accesible from chrome.

**Comment by rgh36167** [comment]
Probably same bug as https://github.com/termux/termux-api/issues/35

**Comment by bumper314** [comment]
I believe this issue is resolved as of Termux v0.49 (2017-05-15)
> Respect --content-type when termux-open/xdg-open is used to open an URL.

**Comment by xeffyr** [comment]
```
termux-open --content-type text/html file:///mnt/sdcard/file.html
```
